File: Javascript/exercicios/ex02.js
// Remova todos os caracteres duplicados da string abaixo:
// "Hello, World!"

export function ex02(text) {
	let letters = text.split("");
	let repeatedLetters = [];
	let result = "";

	for (let i = 0; i < letters.length; i++) {
		if (!repeatedLetters.includes(letters[i])) {
			result += letters[i];
			repeatedLetters.push(letters[i]);
		}
	}

	return result;
}

File: Java/src/Exercicios/Ex02.java
package Exercicios;

import java.util.ArrayList;

// Remova todos os caracteres duplicados da string abaixo:
// "Hello, World!"
public class Ex02 {
  public static String main(String text) {

    int stringSize = text.length();
    ArrayList<String> repeatedLetters = new ArrayList<String>();
    String textArr[] = text.split("");
    String result = "";

    for (int i = 0; i < stringSize; i++) {

      if (!repeatedLetters.contains(textArr[i])) {

        result += textArr[i];
        repeatedLetters.add(textArr[i]);

      }
    }

    return result;
  }
}
